### ISSUE TITLE
Odd visual effects in MacOS 12.0.1 & Safari

### ISSUE THREAD

**Issue by Article19** [pixijs/particle-emitter#167]
MacOS 12.0.1
Safari 15.1 (WebGL 2)
Particles 4.3.1
Pixi 5.3.1 (also happens in 6.2.0)

Particles are behaving & rendering oddly (individual particles appear to get 'abandoned' and no longer update, remaining on-screen). I have tried unsuccessfully to pin it down but could provide screenshots if that would help. This is not happening in other browsers.

Add this to the long (and growing) list of HTML5 issues with MacOS 12 & Safari...

**Comment by andrewstart** [comment]
That's especially weird, as this library is CPU based for particle management, particle update & removal is fully in javasccript.

**Comment by Article19** [comment]
Indeed. Do you have a Monterey system you can test on? Or is it possible to get a copy of 5.0.2 transpiled to ES5 so I can test that? (My users use old gear, so I'm not quite ready to make the jump to ES6, and don't quite have the expertise to do my own transpiling).

**Comment by Article19** [comment]
This is an example of the kind of leftover 'smear' effect I'm seeing (the container is set to 'add')...as though particles are not getting updated to reach an alpha of 0 by the end off their life (which they are set to do)...

![image](https://user-images.githubusercontent.com/12159994/143247393-d7202501-a0f6-47a3-9437-20b8610bbe13.png)

**Comment by Article19** [comment]
Good news! This appears to be fixed in MacOS 12.1.
